arquillian/adapter/src/main/java/org/wildfly/swarm/arquillian/adapter/UberjarSimpleContainer.java

```diff
@@ -355,7 +355,7 @@ private void registerContainerFactory(Archive<?> archive, Class<?> clazz) {
 
     @Override
     public void stop() throws Exception {
-        this.process.stop();
+        this.process.stop(2, TimeUnit.MINUTES); // Because my laptop is slower than yours.
```

Comment:
Technically, this is all that I needed to do .. but then .. I spent so much time cleaning up at other places, I figured it would help at some time.